recipes/simdjson/0.4.6/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class SimdjsonConan(ConanFile):
+    name = "simdjson"
+    version = '0.4.6'
+    description = "Parsing gigabytes of JSON per second"
+    topics = ("conan", "json", "parser", "simd", "format")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lemire/simdjson"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "threads": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True,
+                       'threads': True}
+    _source_subfolder = "source_subfolder"
```

Comment:
namely to cache CMake (configuring twice is expensive)

```suggestion
     _cmake = None
     
    @property
    def _source_subfolder(self):
          return "source_subfolder"
```